road/stats.py

```diff
@@ -0,0 +1,306 @@
+import os
+import pandas as pd
+import numpy as np
+import json
+import yaml
+
+
+def prepare_time_period_dfs(est_df, obs_df, times, combined_df_cols):
+    error = {col: est_df[col] - obs_df[col] for col in times}

```

Comment:
this part seems familiar with some of the code in the other script files. move to a utils.py (or similar) file and import from there